integration/src/test/java/org/mule/test/integration/SchedulerInitialStateTestCase.java

```diff
@@ -39,29 +39,33 @@
 
 import javax.inject.Inject;
 
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+
 import io.qameta.allure.Description;
 import io.qameta.allure.Feature;
 import io.qameta.allure.Story;
-import org.junit.Test;
-import org.junit.runners.Parameterized;
 
 @Feature(SCHEDULER_SERVICE)
 @Story(SOURCE_MANAGEMENT)
 @RunnerDelegateTo(Parameterized.class)
 public class SchedulerInitialStateTestCase extends AbstractIntegrationTestCase {
 
-  private List<Component> recordedOnStartMessageSources = new ArrayList<>();
+  private final List<Component> recordedOnStartMessageSources = new ArrayList<>();
 
   @Inject
   private LazyComponentInitializer lazyComponentInitializer;
 
+  @Inject
+  private ConfigurationComponentLocator configurationComponentLocator;
+
   @Inject
   private TestQueueManager queueManager;
 
   @Parameterized.Parameter
   public boolean lazyInitEnabled;
 
-  @Parameterized.Parameters(name = "{index}: Running tests for {0} (validating XML [{2}]) ")
+  @Parameterized.Parameters(name = "lazyInit: {0}")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.